app/views/users/show_root.html.erb

```diff
@@ -29,14 +29,14 @@
     <hr>
     <div class="bg-cover bg-no-repeat h-full bg-background_img" >
       <div class="pt-5">
-        <div class="grid grid-cols-2 gap-4">
+        <div class="grid md:grid-cols-2 gap-4">
```

Comment:
1. Navbar not coming for unsigned user ?
https://github.com/atifjavaid267/Online-Marketplace-for-Selling-Scrape/pull/77/files#diff-7690d10f7feace11014adc51dbfb2ce2ad66a1f48bf1f26e43e54ec0ab5f2e72R2
If its coming, then code is repeated for the navbars ?